tests/unit/Utopia/Response/Filters/V15Test.php

```diff
@@ -1077,6 +1102,18 @@ public function sessionProvider(): array
                     'providerAccessTokenExpiry' => 1592981250,
                 ],
             ],
+            'empty values' => [
+                [
+                    '$createdAt' => '',
+                    'expire' => '',
```

Comment:
What scenario would cause these to be empty?

Reply:
session.proverAccessTokenExpiry can be an empty string.

Reply:
But `$createdAt` and `expire` can not, right? Let's update the values to reflect that for clarity

Reply:
Updated